gen/aircraft.py

```diff
@@ -724,7 +726,40 @@ def generate_planned_flight(
                     start_type=flight.start_type,
                     at=carrier_group,
                 )
+                return group
             else:
+                # If the flight is an helicopter flight, then prioritize dedicated helipads
+                if flight.unit_type.helicopter:
+                    helipad = cp.get_free_helipad()
+                    if helipad is not None and helipad.static_unit is not None:
+                        group = self._generate_at_group(
+                            name=name,
+                            side=country,
+                            unit_type=flight.unit_type.dcs_unit_type,
+                            count=flight.count,
+                            start_type=flight.start_type,
+                            at=helipad.static_unit,
+                        )
+
+                        # Note : A bit dirty, need better support in pydcs
+                        group.points[0].action = PointAction.FromGroundArea
+                        group.points[0].type = "TakeOffGround"
+                        if flight.start_type != "Cold":
+                            group.points[0].action = PointAction.FromGroundAreaHot
+                            group.points[0].type = "TakeOffGroundHot"
+
+                        helipad.occupied = True
+                        group_generated = True
+
+                        for i in range(flight.count - 1):
+                            helipad = cp.get_free_helipad()
+                            if helipad is not None:
```

Comment:
Should raise an error if this is false. Something's gone wrong if this happens.

Reply:
Resolved in 54124871789535f4c6468abb53c277dd8fd60462 and c4937e95e9e10f94904e742f125ae6c26478ae16